Add newsfragment for #9002 (#9134)
Scille/parsec-cloud: newsfragments/9002.bugfix.rst

> Fix rare freeze in the application when the folder the GUI is displaying gets
> remotely modified.

Reviewer: Word or comma missing? Anyway I'd reword it a little bit
```suggestion
Fix rare freeze when the folder displayed by the GUI is remotely modified.
```
